elements/lisk-p2p/src/utils/misc.ts

```diff
@@ -119,82 +117,6 @@ export const getNetgroup = (address: string, secret: number): number => {
 	return hash(netgroupBytes).readUInt32BE(0);
 };
 
```

Comment:
These functions can remain here and the new `util` functions can also move here, instead of having `util` files on different levels.